dms/models/dms_file.py

```diff
@@ -560,6 +577,39 @@ def _inverse_content(self):
             for vals, ids in updates.items():
                 self.browse(ids).write(dict(vals))
 
+    def _create_model_attachment(self, vals):
+
+        if "default_directory_id" in self._context:
+            default_directory_id = self.env["dms.directory"].search(
+                [("id", "=", self._context["default_directory_id"])]
+            )
```

Comment:
Change is necessary to solved travis
```
            default_directory_id = self.env["dms.directory"].with_user(SUPERUSER_ID).search(
                [("id", "=", self._context["default_directory_id"])]
            )
```
IMO this is a problem with _search function in directory that is not correct (isn't override and is needed) because not apply groups correctlly (not in this PR).

Reply:
Why searching instead of a `browse`?

Reply:
> Why searching instead of a `browse`?

Thanks, update my comments